src/resources/app/pages/PeopleInCourse.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <v-container>
+    <div class="my-10 mx-10">
+      <v-col cols="6">
+        <v-row class="text-primary text-lg" id="title"> Teachers </v-row>
+        <v-row>
+          <v-divider class="bg-primary my-1"></v-divider>
+        </v-row>
+        <v-row
+          v-for="teacher in peoples"
+          v-bind:key="teacher.name"
+          v-if="teacher.role == 'teacher'"
+        >
+          <v-avatar class="bg-primary my-3" id="avatar" size="36"></v-avatar>
+          <div class="ml-3 my-4" id="name">{{ teacher.name }}</div>
+        </v-row>
+        <v-row class="text-primary text-lg" id="title"> Classmates </v-row>
+        <v-row>
+          <v-divider class="bg-primary my-1"></v-divider>
+        </v-row>
+        <v-row
+          v-for="student in peoples"
+          v-bind:key="student.name"
+          v-if="student.role == 'student'"
+        >
+          <v-avatar class="bg-primary my-3" id="avatar" size="36"></v-avatar>
+          <div class="ml-3 my-4" id="name">{{ student.name }}</div>
+        </v-row>
+      </v-col>
+    </div>
+  </v-container>
+</template>
+<script>
+import api from "../../api/people";
+
+export default {
+  data() {
+    return {
+      peoples: [
+        {
+          name: "Placeholder",
+          role: "teacher",
+        },
+      ],
+    };
+  },
+  methods: {
+    async getData() {
+      try {
```

Comment:
It would be nice to create a loading animation while awaiting for api request for better UX. You can utilize icon provided by vuetify and use tailwind for the animation. Example can be seen here:
https://github.com/uasoft-indonesia/badaso-lms-theme/blob/PBI-1-register/src/resources/app/pages/login.vue

Reply:
nice one, let's discuss this in our next sprint planning